qemu/tests/bridge_qinq.py

```diff
@@ -289,7 +288,7 @@ def compare_host_guest_md5sum(name):
         # scp file to guest with L2 vlan tag
         file_size = int(params.get("file_size", "4096"))
         host_path = params.get("host_path", "/tmp/transferred_file")
```

Comment:
Without clean step in host, I think it need use the tmp dir of the framework by force. 
Suggest:
```
from virttest import data_dir
...
host_path = os.path.join(data_dir.get_tmp_dir(), "transferred_file")
```

Without clean step for host path, need

Reply:
Changed it.I will update the test results later